index.html

```diff
@@ -76,132 +82,117 @@ <h3 class="H-title">I'am a Software Developer</h3>
-                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                        <ul class="language-button1">
-                          <li>html</li>
-                          <li>bootstrap</li>
-                          <li>Ruby</li>
-                        </ul>
-                        <button 
-                        type="button"
-                        class="button2">See Project</button>
-                    </div>
-                    <div class="card-work">
-                      <h1 class="papd">Professional Art<br>Printing Data</h1>
-                      <p class="papd-p">Profesional Art Printing Data
-                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                        <ul class="language-button1">
-                          <li>html</li>
-                          <li>bootstrap</li>
-                          <li>Ruby</li>
-                        </ul>
-                        <button 
-                        type="button"
-                        class="button2">See Project</button>
-                    </div>
-                     <div class="card-work">
-                      <h1 class="papd">Professional Art<br>Printing Data</h1>
-                      <p class="papd-p">Profesional Art Printing Data
-                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                        <ul class="language-button1">
-                          <li>html</li>
-                          <li>bootstrap</li>
-                          <li>Ruby</li>
-                        </ul>
-                      <button 
-                      type="button"
-                      class="button2">See Project</button>
-                    </div>
-              </section>
+      <img
+      src="./images/Img.png"
+      class="work__image"
+      alt="project works"
+      data-img="image"
+    />
+      <div class="work__details">
+        <div>
+          <h3 class="work__details__title">Multi-Post Stories</h3>
+        </div>
+       <div>
+        <p class="work__details__p">A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text
+          ever since the 1500s, when an unknown printer took a standard
+          dummy text. A daily selection of privately personalized reads; no
+          accounts or sign-ups required.</p>
+
+       </div>
+       <div>
+        <ul class="single_work_stack">
+          <li>CSS</li>
+          <li>html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+      </ul>
+       </div>
+       <div>
+         
+         <button type="button">See project</button>
+       </div>
+      </div>
+    </div>
+      <section class="somework">
```

Comment:
@prolajumokeoni ,
This is on a separate branch, we were instructed to do the transition and animation on a separate branch.

Reply:
- [ ] As suggested by the previous reviewer, Kindly add hover effects to the cards in this section, when the project is hovered on with the mouse the description and dark opacity disappears and a See Project button appears at the bottom of the image.
The hover effect when implemented should look like your first card do this for all cards, kindly make all cards to be the same and add this cool feature, this will make your design look like the Figma design.